app/controllers/application_controller.rb

```diff
@@ -17,6 +18,10 @@ def after_sign_in_path_for(resource)
 
   private
 
+  def current_event
+    @current_event ||= Event.find_by(id: session[:event_id]) if session[:event_id]
+  end
+
```

Comment:
Why is there a helper for this in both application_controller and application_helper? Seems like duplicate code?